lib/item.rb

```diff
@@ -1,3 +1,8 @@
 class Item
+  attr_reader :name, :price
 
+  def initialize(item_info)
+    @name = item_info[:name]
+    @price = item_info[:price]
```

Comment:
The interaction pattern asks to clean the string (remove the $) and convert to a float